Lib/_colorize.py

```diff
@@ -0,0 +1,45 @@
+import io
+import os
+import sys
+
+COLORIZE = True
+
+
+class ANSIColors:
+    BOLD_GREEN = "\x1b[1;32m"
+    BOLD_MAGENTA = "\x1b[1;35m"
+    BOLD_RED = "\x1b[1;31m"
+    GREEN = "\x1b[32m"
+    GREY = "\x1b[90m"
+    MAGENTA = "\x1b[35m"
+    RED = "\x1b[31m"
+    RESET = "\x1b[0m"
+    YELLOW = "\x1b[33m"
+
+
+def can_colorize():
```

Comment:
```suggestion
def can_colorize() -> bool:
```

Consider adding typing.

Reply:
In general our policy is not to add type hints to standard-library modules in the `Lib/` directory (there are some exceptions, but they're all pretty specific). I think we still have yet to add some formal documentation around this in the devguide, but you can see some rationales here:
- https://github.com/python/cpython/pull/108125#discussion_r1313654070
- https://discuss.python.org/t/type-annotations-in-the-stdlib/21487

Reply:
Thank you for this explanation, I made another issue to formalize the problem.

Reply:
Thanks so much @Privat33r-dev!